src/Nethermind/Nethermind.Mev/Data/SelectorType.cs

```diff
@@ -15,11 +15,10 @@
 //  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
 // 
 
-namespace Nethermind.Mev.Test
+namespace Nethermind.Mev.Data
 {
-    public enum TailGasType
+    public enum SelectorType
```

Comment:
Another one?

Reply:
not really needed as of right now, tho it will be useful for future versions